Makefile

```diff
@@ -30,8 +30,6 @@ assets:
 	npm install && \
 	cd ${CWD}
 	${POOTLE_CMD} compilejsi18n
-	# The site language en-us is not compiled by default, so handle it specially
-	${POOTLE_CMD} compilejsi18n -l en-us
```

Comment:
> where is the symbolic link?

[It's here.](https://github.com/julen/pootle/blob/32160c5dd1b5145abbdb2301fd45374c46731756/pootle/locale/en_US)